apps/editing-toolkit/editing-toolkit-plugin/tags-education/src/add-tags-education-link.js

```diff
@@ -28,7 +28,7 @@ const addTagsEducationLink = createHigherOrderComponent( ( PostTaxonomyType ) =>
 						// However, if we have any string without translation in `default` domain, the configure block will be overwritten by empty locale data
 						// so that we cannot get the correct locale from the @wordpress/i18n locale data.
 						// Also, the format of current locale slug is not ISO 639 and it makes localizeUrl not work correctly.
-						window.tagsEducationLocale
+						window.ETK_I18N_LOCALE_CODE
```

Comment:
@arthur791004, could you let me know what the current status is for this one? Can we just delete these lines altogether? If not, did the [linked PR](https://github.com/Automattic/wp-calypso/pull/55527) change anything about how we should handle this?

Reply:
This one is the related ticket: https://github.com/Automattic/wp-calypso/issues/60111.

The problem is that the domain configuration would be overridden if we call `setLocaleData` multiple times. However, we don't have language data in the untranslated JSON file (e.g af.json) so the language (`i18n.getLocaleData()?.[ '' ]?.language`) in the locale data would be empty and we cannot get the correct one. Thus, I make a change to add the `addLocaleData` function in https://github.com/WordPress/gutenberg/pull/37704 to avoid overriding the domain configuration and try to replace `setLocaleData` with it. But it's in the WordPress Core function (See https://github.com/WordPress/WordPress/blob/48aa21e9d9894696de4102783f17e81441c8e134/wp-includes/class.wp-scripts.php#L610), so I [file a ticket](https://core.trac.wordpress.org/ticket/54858#ticket) and it's still in the discussion because of the backward compatibility concerns.